app/models/user.rb

```diff
@@ -59,6 +60,10 @@ def password_reset_expired?
     reset_sent_at < Settings.time_expired.hours.ago
   end
 
+  def feed
+    Micropost.where("user_id = ?", id)
```

Comment:
`where("user_id = ?", id)`
you should research `scope in rails` then defind scope for this code